testsuite/features/support/cobbler_test.rb

```diff
@@ -67,30 +81,46 @@ def profile_create(name, distro, location)
     profile_id
   end
 
-  def profile_exists(name)
-    exists('profiles', 'name', name)
+  # every system needs at least a name and a profile
+  def system_create(name, profile)
+    begin
+      system_id = @server.call('new_system', @token)
+    rescue
+      raise 'creating system failed.' + $ERROR_INFO.to_s
+    end
+    begin
+      @server.call('modify_system', system_id, 'name', name, @token)
+      @server.call('modify_system', system_id, 'profile', profile, @token)
+    rescue
+      raise 'modifying system failed.' + $ERROR_INFO.to_s
+    end
+    begin
+      @server.call('save_system', system_id, @token)
+    rescue
+      raise 'saving system failed.' + $ERROR_INFO.to_s
+    end
+    system_id
   end
 
-  def system_exists(name)
-    exists('systems', 'name', name)
+  def system_remove(name)
+    raise "system cannot be found. #{$ERROR_INFO}" unless system_exists(name)
+    begin
+      @server.call('remove_system', name, @token)
+    rescue
+      raise "deleting system failed. #{$ERROR_INFO}"
```

Comment:
Nitpick, sometimes we use string format, other we don't.
I mean, sometimes we do "xxxx #{yyy}" others 'xxxx' + yyy.to_s

Let's be consistent in our code style.

Reply:
This is intentional since

`'xxxx' + yyy.to_s` is an offence according to RuboCop.

We already have this card to improve the `CobblerTest` Class: https://github.com/SUSE/spacewalk/issues/17418

Reply:
Good for the Cop, because the string formatter is also my prefered way :)